modules/ROOT/pages/advanced-mule-concepts.adoc

```diff
@@ -0,0 +1,17 @@
+= Advanced Mule Concepts
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: advanced, mule, concepts, runtime, engine
+
+This section covers some advanced Mule topics that will be useful once you are familiar with the basics of Mule application development and Mule runtime engine concepts.
+
+Learn about the xref:execution-engine.adoc[Execution Engine] to understand how the Mule runtime engine processes the operations in a Mule application’s flow, and understand how to change the thread pools configuration to optimize performance.
+
+The xref:about-classloading-isolation.adoc[Classloading Isolation] mechanism implemented in Mule 4 addresses potential issues caused by the Java Virtual Machine when conflicting versions of the same JAR files exist in the same location.
+
+Mule 4 supports xref:distributed-file-polling.adoc[Distributed File Polling], which allows connectors to read files from certain directories as they are created, and it also makes it possible to poll files in all cluster nodes.
+
+You can check the xref:mule-4-api-javadoc.adoc[Javadocs] for the Mule Kernel, the Community Edition of the Mule runtime engine, to see definitions for classes, fields and their data types.
+
+MuleSoft also provides a list of xref:third-party-software-in-mule.adoc[Third-party software] that is included as part of the source code, examples, default services or as dependencies.
```

Comment:
`provides` doesn't sound good, since the libs bundled in the runtime are for internal use of the runtime, while `provide` would be interpreted that it is something that you may use from an app.